apps/dcos_net/test/dcos_net_mesos_listener_tests.erl

```diff
@@ -314,6 +321,24 @@ hello_overlay_host_vip(Tasks) ->
         state => running
     }, maps:get(TaskId, Tasks)).
 
+%%%===================================================================
+%%% Pod Tasks Tests
+%%%===================================================================
+
+pod_tasks_foo(Tasks) ->
+    TaskId = <<"app.instance-caff1565-d0db-11e8-aa23-70b3d5800002.foo">>,
+    ?assertEqual(#{
+        name => <<"app">>,
+        framework => <<"marathon">>,
+        agent_ip => {172, 17, 0, 3},
+        task_ip => [{9, 0, 0, 2}],
+        state => running
+    }, maps:get(TaskId, Tasks)).
```

Comment:
you might have to update this test or #95 depending on the sequence in which you commit the PRs